### ISSUE TITLE
Unit tests fail with missing pandas.Int64Index

### ISSUE THREAD

**Issue by mplpl** [IBM/nzpyida#20]
```
FAILED test_frame.py::Test_OpenDataFrameObject::test_idadf_attr_index - AttributeError: module 'pandas' has no attribute 'Int64Index'
```

**Comment by mplpl** [comment]
Here is what I read in pandas documentation for Int64Index:
```
Deprecated since version 1.4.0: In pandas v2.0 Int64Index will be removed and NumericIndex used instead. Int64Index will remain fully functional for the duration of pandas 1.x.
```

**Comment by mplpl** [comment]
Fixed in 0.3.3
